src/test/java/it/gov/pagopa/arc/config/OAuth2LoginConfigTest.java

```diff
@@ -62,10 +64,11 @@ void givenURLWhenWithoutAccessTokenThenRedirectToLogin() throws Exception {
     }
 
     @Test
-    void givenInsecurityURLWhenCallEndpointThenNoRedirect() throws Exception {
-        mockMvc.perform(MockMvcRequestBuilders.get("/token/assistance")
+    void givenSecurityURLWhenCallEndpointThenNoRedirect() throws Exception {
+        MvcResult result = mockMvc.perform(MockMvcRequestBuilders.get("/token/assistance")
                         .param("userEmail", "someone@email.com"))
-                .andExpect(status().isOk());
+                .andExpect(status().is4xxClientError()).andReturn();
+        Assertions.assertNotEquals(302,result.getResponse().getStatus());
```

Comment:
the second assertion "not equal 302" is irrelevant since the first assertion is "equal 4xx", do you think?

Reply:
removed